### ISSUE TITLE
DDL: low hanging fruit content design tweaks

### ISSUE THREAD

**Issue by jacobworrell** [department-of-veterans-affairs/vets-website#22633]
- [x] Page title needs to be changed. The page title is "Track Claims | Veterans Affairs" when it should be something like "Your VA Claim Letters | Veterans Affairs"

- [ ] Might be worth including in the description under the heading that letters are organized with the most recent first.

- "Letters are sorted with the most recent first." But let's check w/ Josh on the relative importance of this one.

## Tasks
- [x] Change page title to `Your VA Claim Letters | Veterans Affairs`
- [x] There is no content displayed when a user doesn't have any letters
- [x] Users get errors in the console when they don't have any letters
- [x] Dates in the `Letters dated ...` text are off by a day
- [x] The entire application reloads when the user clicks the `Get your claim letters` link
